_sass/main.scss

```diff
@@ -1,6 +1,74 @@
-// import global settings
-//@import 'global-settings';
+// Import site settings
+@import 'settings';
 
 // import vanilla themes
 @import "../node_modules/vanilla-brochure-theme/scss/theme";
 @include vanilla-brochure-theme;
+
+// Import modules
+@import 'patterns_navigation';
+@include cu-p-navigation;
+
+@import 'patterns_footer';
+@include cu-p-footer;
+
+body {
+  background: $color-light;
+  background-image: url("https://assets.ubuntu.com/sites/ubuntu/latest/u/img/backgrounds/image-background-paper.png");
```

Comment:
This should fix CI:
Please use `#{$assets-path}f8a323a7-image-background-paper.png`